util/swm-ssh/swm-ssh.lisp

```diff
@@ -0,0 +1,39 @@
+;;;; swm-ssh.lisp
+
+(defpackage #:swm-ssh
+  (:use #:cl #:stumpwm)
+  (:export :*swm-ssh-default-term*
+           :*swm-ssh-config-path*)
```

Comment:
This is a nit, but it would be more consistent to use uninterned symbols here.
```suggestion
  (:export #:*swm-ssh-default-term*
           #:*swm-ssh-config-path*)
```